<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsBrowserMail</class>
 <widget class="QWidget" name="SettingsBrowserMail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>330</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="m_tabBrowserProxy">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_tabExternalBrowser">
      <attribute name="title">
       <string>External web browser</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_17">
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="m_grpCustomExternalBrowser">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If unchecked, then default system-wide web browser is used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Use custom external web browser</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_12">
          <item row="0" column="0">
           <widget class="QLabel" name="m_lblExternalBrowserExecutable">
            <property name="text">
             <string>Web browser executable</string>
            </property>
            <property name="buddy">
             <cstring>m_txtExternalBrowserExecutable</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLineEdit" name="m_txtExternalBrowserExecutable">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Executable file of web browser</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnExternalBrowserExecutable">
              <property name="text">
               <string>&amp;Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_lblExternalBrowserArguments">
            <property name="text">
             <string>Parameters</string>
            </property>
            <property name="buddy">
             <cstring>m_txtExternalBrowserArguments</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="m_txtExternalBrowserArguments">
            <property name="placeholderText">
             <string>Parameters passed to executable</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_lblExternalBrowserPreset">
            <property name="text">
             <string>Use sample arguments for</string>
            </property>
            <property name="buddy">
             <cstring>m_cmbExternalBrowserPreset</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="m_cmbExternalBrowserPreset">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>Select browser</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Note that &quot;%1&quot; (without quotation marks) is placeholder for URL of selected message.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="m_checkOpenLinksInExternal">
         <property name="text">
          <string>Always open links from simple internal text browser in external web browser</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>External e-mail client</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_22">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="m_grpCustomExternalEmail">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If unchecked, then default system-wide web browser is used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Use custom external e-mail client</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_21">
          <item row="0" column="0">
           <widget class="QLabel" name="m_lblExternalEmailExecutable">
            <property name="text">
             <string>E-mail client executable</string>
            </property>
            <property name="buddy">
             <cstring>m_txtExternalEmailExecutable</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLineEdit" name="m_txtExternalEmailExecutable">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Executable file of e-mail client</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnExternalEmailExecutable">
              <property name="text">
               <string>&amp;Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_lblExternalEmailArguments">
            <property name="text">
             <string>Parameters</string>
            </property>
            <property name="buddy">
             <cstring>m_txtExternalEmailArguments</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="m_txtExternalEmailArguments">
            <property name="placeholderText">
             <string>Parameters passed to executable</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_lblExternalEmailPreset">
            <property name="text">
             <string>Use sample arguments for</string>
            </property>
            <property name="buddy">
             <cstring>m_cmbExternalEmailPreset</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="m_cmbExternalEmailPreset">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>Select client</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="m_lblExternalEmailInfo">
            <property name="text">
             <string>Placeholders:
 • %1 - title of selected message,
 • %2 - body of selected message.</string>
            </property>
            <property name="indent">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>External tools</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="m_btnAddTool">
           <property name="text">
            <string>Add external tool</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnDeleteTool">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete selected external tool</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="m_lblToolInfo">
         <property name="text">
          <string>On this page, you can setup a list of external tools which can open URLs of selected messages.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTreeWidget" name="m_listTools">
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabProxy">
      <attribute name="title">
       <string>Proxy</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="m_lblProxyType">
         <property name="text">
          <string extracomment="Proxy server type.">Type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_cmbProxyType</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="m_cmbProxyType"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_lblProxyHost">
         <property name="text">
          <string>Host</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_txtProxyHost</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLineEdit" name="m_txtProxyHost">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Hostname or IP of your proxy server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_lblProxyPort">
           <property name="text">
            <string>Port</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>m_spinProxyPort</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_spinProxyPort">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_lblProxyUsername">
         <property name="text">
          <string>Username</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_txtProxyUsername</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="m_txtProxyUsername">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Your username for proxy server authentication</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="m_lblProxyPassword">
         <property name="text">
          <string>Password</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_txtProxyPassword</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="m_txtProxyPassword">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
         <property name="placeholderText">
          <string>Your password for proxy server authentication</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="m_checkShowPassword">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Display password</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="m_lblProxyInfo">
         <property name="text">
          <string>Note that these settings are applied only on newly established connections.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_tabBrowserProxy</tabstop>
  <tabstop>m_checkOpenLinksInExternal</tabstop>
  <tabstop>m_grpCustomExternalBrowser</tabstop>
  <tabstop>m_txtExternalBrowserExecutable</tabstop>
  <tabstop>m_btnExternalBrowserExecutable</tabstop>
  <tabstop>m_txtExternalBrowserArguments</tabstop>
  <tabstop>m_cmbExternalBrowserPreset</tabstop>
  <tabstop>m_grpCustomExternalEmail</tabstop>
  <tabstop>m_txtExternalEmailExecutable</tabstop>
  <tabstop>m_btnExternalEmailExecutable</tabstop>
  <tabstop>m_txtExternalEmailArguments</tabstop>
  <tabstop>m_cmbExternalEmailPreset</tabstop>
  <tabstop>m_listTools</tabstop>
  <tabstop>m_btnAddTool</tabstop>
  <tabstop>m_btnDeleteTool</tabstop>
  <tabstop>m_cmbProxyType</tabstop>
  <tabstop>m_txtProxyHost</tabstop>
  <tabstop>m_spinProxyPort</tabstop>
  <tabstop>m_txtProxyUsername</tabstop>
  <tabstop>m_txtProxyPassword</tabstop>
  <tabstop>m_checkShowPassword</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
